ocs_ci/deployment/ingress_node_firewall.py

```diff
@@ -106,6 +120,70 @@ class IngressNodeFirewallInstaller(object):
 
     def __init__(self):
         self.namespace = constants.INGRESS_NODE_FIREWALL_NAMESPACE
+        self.source = constants.OPERATOR_CATALOG_SOURCE_NAME
+
+    def icsp_brew_registry_exists(self):
```

Comment:
As we discussed offline, I suggest to open issue to move the logic of brew registry to different module and reuse it in other places as well to not repeat the code. This way we can move first code out of this modul to the other, and verify only this change here. The other places - we can replace in separate PRs and verify them separately to know we didn't break other things.

Reply:
For now, I've at least moved the code related to qe-app-registry catalog to new module: 6cad0f41c911dc41a1a14b2c9a6ced49b41897c1